### DISCUSSION TITLE
Screenflick 3

### DISCUSSION THREAD

**Discussion by pindab0ter** [elrumo/macOS_Big_Sur_icons_replacements#1322] [Icon request] [answered]
https://www.araelium.com/screenflick-mac-screen-recorder

Current:
<img width="512" alt="Screenflick" src="https://user-images.githubusercontent.com/5128166/140503499-b12665c7-754a-4700-9258-3d0526467b2c.png">

[Screenflick.icns.zip](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/files/7484453/Screenflick.icns.zip)

**Comment by bunkerdesign** [comment, marked as answer]
[Screenflick 3.icns.zip](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/files/7603786/Screenflick.3.icns.zip)

**Reply by pindab0ter** [reply]
That can't have been an easy one, but it looks great!

**Reply by bunkerdesign** [reply]
You are welcome! :)
